lstchain/tools/lstchain_create_calibration_file.py

```diff
@@ -2,103 +2,77 @@
 Extract flat field coefficients from flasher data files.
 """
 import numpy as np
-from traitlets import Dict, List, Unicode, Float, Bool
-
-
-from ctapipe.core import Provenance, traits
-from ctapipe.io import HDF5TableWriter
-from ctapipe.core import Tool
-from ctapipe.io import EventSource
 from ctapipe.containers import PixelStatusContainer
+from ctapipe.core import Provenance, Tool, traits
+from ctapipe.io import EventSource, HDF5TableWriter
+
 from lstchain.calib.camera.calibration_calculator import CalibrationCalculator
 from lstchain.calib.camera.r0 import CameraR0Calibrator
 from lstchain.io.lstcontainers import LSTEventType
-
-__all__ = [
-    'CalibrationHDF5Writer'
-]
+from tqdm.autonotebook import tqdm
 
 
 class CalibrationHDF5Writer(Tool):
-    """
-     Tool that generates a HDF5 file with camera calibration coefficients.
-     This is just an example on how the monitoring containers can be filled using
-     the calibration Components in calib/camera.
-     This example is based on an input file with pedestal and flat-field events
-
-     For getting help run:
-     python calc_camera_calibration.py --help
-
-     """
 
     name = "CalibrationHDF5Writer"
     description = "Generate a HDF5 file with camera calibration coefficients"
 
+    one_event = traits.Bool(False, help="Stop after first calibration event").tag(config=True)
 
-    one_event = Bool(
-        False,
-        help='Stop after first calibration event'
+    output = traits.Path(
+        help="Name of the output file",
+        directory_ok=False,
+        default_value="calibration.hdf5"
     ).tag(config=True)
 
-    output_file = Unicode(
-        'calibration.hdf5',
-        help='Name of the output file'
-    ).tag(config=True)
-
-    log_file = Unicode(
-        'None',
-        help='Name of the log file'
+    progress_bar = traits.Bool(
+        help="show progress bar during processing",
+        default_value=True,
     ).tag(config=True)
 
     calibration_product = traits.create_class_enum_trait(
-       CalibrationCalculator,
-        default_value='LSTCalibrationCalculator'
+        CalibrationCalculator, default_value="LSTCalibrationCalculator"
     )
-
-    r0calibrator_product =traits.create_class_enum_trait(
-        CameraR0Calibrator,
-        default_value='NullR0Calibrator'
+    r0calibrator_product = traits.create_class_enum_trait(
+        CameraR0Calibrator, default_value="NullR0Calibrator"
     )
 
-    aliases = Dict(dict(
-        input_file='EventSource.input_url',
-        output_file='CalibrationHDF5Writer.output_file',
-        log_file='CalibrationHDF5Writer.log_file',
-        max_events='EventSource.max_events',
-        pedestal_file= 'LSTR0Corrections.pedestal_path',
-        calibration_product='CalibrationHDF5Writer.calibration_product',
```

Comment:
Please, keep the pedestal_file alias, this is not making big damage and can be useful for fast tests

Reply:
I can easily bring back `pedestal_file` alias, but I do not see where in the code this pedestal file could be used now. I guess you mean it could be used in a potential feature that it is still not implemented. In that case, and since it will not be used if we bring the alias back, shouldn't we just add it in the PR implementing that *now missing* feature?

Reply:
You are right, in this version it is no more useful. Forget it.